### PR TITLE
Build eks-anywhere-packages and include it in releases

### PR DESCRIPTION
**Pull request by ewollesen**
Trying to get eks anywhere packages included in the build/release process.

### CODE REVIEW

File: release/staging-build.yml [hunk 1]
```diff
@@ -205,3 +205,9 @@ batch:
     depend-on:
     - cri_tools
     - etcdadm
+  - identifier: eks-anywhere-packages
```
Comment (abhay-krishna) [inline release/staging-build.yml:208]: Identifier names cannot have dashes.
```suggestion
  - identifier: eks_anywhere_packages
```

(I know right 🙄)

File: release/staging-build.yml [hunk 2]
```diff
@@ -205,3 +205,21 @@ batch:
     depend-on:
     - cri_tools
     - etcdadm
+  - identifier: eks_anywhere_packages
```
Comment (abhay-krishna) [inline release/staging-build.yml:208]: let's reorder to put this at the bottom, and add a depend-on field to the packages build
```yaml
depend-on:
- harbor
- hello_eks_anywhere
```

line 205 has an example of adding such a clause, and its indentation.

Comment (ewollesen) [reply release/staging-build.yml:208]: Oh shoot, right, obviously I forgot that part. Thank you.

Comment (abhay-krishna) [reply release/staging-build.yml:208]: this is because identifiers used in the `depend-on` section need to be defined before being used.

Comment (ewollesen) [reply release/staging-build.yml]: Heh, I was kinda wondering if there were ordering requirements.

**Review by abhay-krishna — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**